bin/compile

```diff
@@ -40,7 +40,18 @@ BUILD_DIR=$1
 CACHE_DIR=$2
 ENV_DIR=$3
 BAZEL_BUILD_PATH=$(cat ${ENV_DIR}/BAZEL_BUILD_PATH)_deploy.jar
+USE_BUILD_CACHE=$(cat ${ENV_DIR}/USE_BUILD_CACHE)
 
+echo "-----> BUILD_DIR: ${BUILD_DIR}"
+echo "-----> CACHE_DIR: ${CACHE_DIR}"
+echo "-----> ENV_DIR: ${ENV_DIR}"
+echo "-----> BAZEL_BUILD_PATH: ${BAZEL_BUILD_PATH}"
+echo "-----> USE_BUILD_CACHE: ${USE_BUILD_CACHE}"
+
+if [[ -n $USE_BUILD_CACHE ]]; then
+  BUILD_CACHE_BUCKET=$(cat ${ENV_DIR}/BUILD_CACHE_BUCKET)
+  GOOGLE_STORAGE_CREDENTIAL_FILE_PATH=$BUILD_DIR/$(cat ${ENV_DIR}/GOOGLE_STORAGE_CREDENTIAL_FILE_PATH)
```

Comment:
Prefer prefix consistency so the env vars are listed together by Heroku.
Ex. `BUILD_CACHE_CREDENTIALS_PATH`

Which goes well with `BUILD_CACHE_PATH` or `BUCKET` is okay too!